packages/atlas/src/views/studio/MyChannelView/tabs/GeneralTab/StudioChannelGeneralTab.tsx

```diff
@@ -210,27 +235,41 @@ export const GeneralTab = ({ actionBarPortal }: { actionBarPortal: RefObject<HTM
           </InputsWrapper>
         </EntitySettingTemplate>
       </form>
```

Comment:
I mentioned this in my previous review. Let's add padding-bottom on the form(I think this should be 52px according to the designs), the spacing feels awkward to me currently